classes/patterns/class-sideimagewithtextandcta.php

```diff
@@ -34,13 +34,16 @@ public static function get_config( $params = [] ): array {
 		$background_color     = $params['background_color'] ?? null;
 		$background_attribute = $background_color ? ',"backgroundColor":"' . $background_color . '"' : '';
 		$background_classes   = $background_color ? 'has-' . $background_color . '-background-color has-background' : '';
+		$alignfull            = $params['alignfull'] ?? false;
+		$alignment_class      = $alignfull ? 'alignfull' : '';
```

Comment:
I don't think it's necessary to set it to `alignwide` if it's not full width, it seems to be added by default!